### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by hocnv90** [wanasit/katakana-seq2seq#3]
It should be 'categorical_crossentropy'? Right?

**Comment by wanasit** [comment]
Thank you. You are right. I should have used the 'categorical_crossentropy', instead of binary_crossentropy. I will fix this soon.

FYI. The reason it works is that, at least in Tensorflow backend, Keras's binary_crossentropy and categorical_crossentropy are actually backed by the same implementation (`tf.nn.softmax_cross_entropy_with_logits`).
